ultra-moon-hawlucha/route.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Ultra Moon Hawlucha Any%",

```

Comment:
Please don't include the name of the game in the title - Ranger will show it already via the `game` property.